internal/controllers/machine/machine_controller.go

```diff
@@ -115,7 +115,10 @@ func (r *Reconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, opt
 		handler.EnqueueRequestsFromMapFunc(clusterToMachines),
 		// TODO: should this wait for Cluster.Status.InfrastructureReady similar to Infra Machine resources?
 		predicates.All(ctrl.LoggerFrom(ctx),
-			predicates.ClusterUnpaused(ctrl.LoggerFrom(ctx)),
+			predicates.Any(ctrl.LoggerFrom(ctx),
+				predicates.ClusterUnpaused(ctrl.LoggerFrom(ctx)),
+				predicates.ClusterUpdateNotPaused(ctrl.LoggerFrom(ctx)),
```

Comment:
Isn't this filter a superset of the filter above?

Reply:
yep, but that's not relevant anymore as I've introduced another predicate.